politeiawww/user/cockroachdb/cockroachdb.go

```diff
@@ -741,6 +744,122 @@ func (c *cockroachdb) RegisterPlugin(p user.Plugin) error {
 	return nil
 }
 
+func (c *cockroachdb) FetchHistories24h(recipients []string) ([]user.EmailHistory24h, error) {
```

Comment:
Not sure what practice you follow, I believe it is worth to put a comment only when it adds any meaningful information,
and here the description I've added to the interface this `func` implements fully covers it (nothing new to add),
so I've omitted it (I believe some linters follow that rule as well),

do you want ?
- to copy paste the description from the `interface`
- or just leave an ugly comment like this `// FetchHistories24h ` 

also, if that's a strict requirement, probably it would make sense to enforce it by a linter.

Reply:
Every exported (capitalized) name in a program should have a doc comment. 
https://golang.org/doc/effective_go#commentary